lib/mvi/messages/find_candidate_message.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+require 'ox'
+require_relative 'message_builder_concern'
+
+module MVI
+  module Messages
+    # Builds an MVI SOAP XML message.
+    #
+    # = Usage
+    # Call the .build passing in the candidate's given and family names, dob, and ssn.
+    #
+    # Example:
+    #  dob = Time.new(1980, 1, 1).utc
+    #  message = MVI::Messages::FindCandidateMessage.new(['John', 'William'], 'Smith', dob, '555-44-3333').to_xml
+    #
+    class FindCandidateMessage
+      include MVI::Messages::MessageBuilder
+      EXTENSION = 'PRPA_IN201305UV02'
+
+      attr_reader :given_names, :family_name, :dob, :ssn
+
+      validate :validate_types
+      validates_format_of :ssn, with: /\d{3}-\d{2}-\d{4}/
+
+      def initialize(given_names, family_name, dob, ssn)
```

Comment:
@plusjeff RE: middle name - MVI uses _n_ 'given names' for first, middle, and anything other than family so this will be an array.